table/server/common/src/main/java/alluxio/table/common/udb/PathTranslator.java

```diff
@@ -27,7 +30,14 @@
 public class PathTranslator {
   private static final Logger LOG = LoggerFactory.getLogger(PathTranslator.class);
 
-  private BiMap<String, String> mPathMap;
+  private final BiMap<AlluxioURI, AlluxioURI> mPathMap;
+
+  private static final String SCHEME_AUTHORITY_PREFIX;
+
+  static {
+    SCHEME_AUTHORITY_PREFIX =
+        ConfigurationUtils.getSchemeAuthority(ServerConfiguration.global());
+  }
```

Comment:
why not to consolidate the code?

```suggestion
  private static final String SCHEME_AUTHORITY_PREFIX =
      ConfigurationUtils.getSchemeAuthority(ServerConfiguration.global());

```

Reply:
My understanding is that static items are initialized when the class is loaded. Can we know for sure that `ServerConfiguration.global()` will return something meaningful when `PathTranslator` is loaded?